src/util/LinkButton.js

```diff
@@ -1,24 +1,8 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styled from 'styled-components';
+import {StyledLinkButton} from '../components/common/styles';
 import {withRouter} from 'react-router';
```

Comment:
Maybe I didn't explain this correctly but I meant to style the LinkButton in the file in which you import it. To do that you'd do something like

import LinkButton from '../LinkButton'

const StyledLinkButton = styled(LinkButton)` ... (styles) `;

etc.

The default LinkButton would thus have no styling attatched to it.

Reply:
Ohh I see what you're saying. I still don't fully understand the reasoning for this but I am assuming it's to keep the 'Styled' in front of all the Styled components' names for consistency reasons.

Reply:
I guess it could also be to have different styling of login vs sign up